docs/map-making/upgrading-1-8-maps.md

```diff
@@ -1,88 +1,147 @@
+
+### New XML settings:
+
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
+
+`<triplea minimumVersion="*"/>`:`<triplea minimumVersion="1.9.0.0"/>`
+
+(The "*" means whatever)
+
+### Not already deprecated XML changes:
 
-`<triplea minimumVersion="1.9.0.0"/>`
+_(these are the main changes on game file items that were not deprecated, so you normally need to change these only, within the game file)_
 
-### Not already deprecated xml changes:
-_(you normally need to change these only)_
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
 
 `attatchment`:`attachment`
+
 `Attatchment`:`Attachment`
+
 `attatchTo=`:`attachTo=`
+
 `="isImpassible"`:`="isImpassable"`
+
 `="conditionType" value="XOR"`:`="conditionType" value="1"`
-`="turns":="rounds"`
+
+`="turns"`:`="rounds"`
+
 `="Battleships repair at end of round"`:`="Units Repair Hits End Turn"`
+
 `="Battleships repair at beginning of round"`:`="Units Repair Hits Start Turn"`
+
 `="Naval Bombard Casualties Return Fire Restricted"`:`="Naval Bombard Casualties Return Fire"`
 
-### Special code xml changes for the above changes:
-_(you probably don't have these)_
+### Special code XML changes for the above changes:
+
+_(these are cases of advanced coding, so you probably don't have these)_
+
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
 
 `value="conditionType" count="XOR"`:`value="conditionType" count="1"`
+
 `value="conditionType" count="-reset-XOR"`:`value="conditionType" count="-reset-1"`
 
-### Already deprecated but effective xml changes:
-_(you shouldn't have had these in your games already)_
+### Already deprecated but effective XML changes:
+
+_(these items were already deprecated, so you shouldn't have had these in your games already)_
+
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
 
 `="isTwoHit" value="true"`:`="hitPoints" value="2"`
+
 `="isTwoHit" value="false"`:`="hitPoints" value="1"`
+
 `="occupiedTerrOf"`:`="originalOwner"`
+
 `="victoryCity" value="true"`:`="victoryCity" value="1"`
+
 `="victoryCity" value="false"`:`="victoryCity" value="0"`
 
-### Already deprecated and ineffective xml deletions, for all the respective attachments:
-_(you shouldn't have had these in your games already and, if you did, they were doing nothing)_
+### Already deprecated and ineffective XML deletions, for all the respective attachments:
```

Comment:
" ineffective XML deletions, " is a bit odd. I'm not actually sure what that means.

Reply:
Well, it is the code being deleted that it was deprecated and ineffective. I guess it could be explained better, but this is the same compact format I used in the other places for titles. I would need to make all titles longer, I think.

Would you change "Already deprecated and ineffective XML deletions, for all the respective attachments" with "Deletions of already deprecated and ineffective XML items, for all the respective attachments"? If I do this, I would also change most other tiles, making them clearer but less compact. No problem.